frontend/public/kubevirt/components/vm/vm-detail.jsx

```diff
@@ -30,15 +34,30 @@ import { Disk } from '../disk';
 import { NodeLink, LoadingInline } from '../../../components/utils';
 import { menuActions } from './menu-actions';
 
-const VmiEvents = ({obj: vm}) => {
-  const vmi = {
-    kind: VirtualMachineInstanceModel.kind,
-    metadata: {
-      name: vm.metadata.name,
-      namespace: vm.metadata.namespace,
-    },
+const VmEvents = ({ obj: vm }) => {
+  const vmObj = {
+    name: getName(vm),
+    namespace: getNamespace(vm),
+  };
+  const vmFilter = obj => _.isMatch(obj, {...vmObj, kind: VirtualMachineModel.kind});
+  const vmiFilter = obj => _.isMatch(obj, {...vmObj, kind: VirtualMachineInstanceModel.kind});
+  const launcherPodFilter = ({ kind, namespace, name }) =>
+    kind === PodModel.kind && namespace === getNamespace(vm) && name.startsWith(`${VIRT_LAUNCHER_POD_PREFIX}${getName(vm)}-`);
+  const importerPodFilter = ({ kind, namespace, name }) => {
+    // importer pod example importer-<diskName>-<vmname>-<generatedId>
+    // note: diskName may contain '-' which means pod name should have at least 4 parts
+    if (kind === PodModel.kind && namespace === getNamespace(vm) && name.startsWith('importer')) {
+      const dashIndex = name.lastIndexOf('-');
+      if (dashIndex > 0) {
+        const vmName = name.slice(0, dashIndex);
```

Comment:
last nit: we could also exclude the part with the first two dashes

Reply:
`importer-` part is now excluded.